app/employee-app/backend/modules/database/db_queries.bal

```diff
@@ -51,6 +51,19 @@ isolated function getCommonEmployeeQuery() returns sql:ParameterizedQuery => `
         hris_employment_type het ON e.employee_employment_type_id = het.employment_type_id
 `;
 
+# Build query to retrieve the distinct employee locations.
+#
+# + employeeStatuses - List of employee statuses to consider
+# + return - sql:ParameterizedQuery - Select query for the employee table
```

Comment:
IMentioning the data type in the documentation for parameters and return values isn't necessary, as the generated documentation infers this from the method signature.
```suggestion
# + return - Select query for the employee table
```